Project.toml

```diff
@@ -8,6 +8,9 @@ DiffEqBase = "2b5f629d-d688-5b77-993f-72d75c75574e"
 Flux = "587475ba-b771-5e3f-ad9e-33799f191a9c"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+StochasticDiffEq = "789caeaf-c7a9-5a7d-9973-96adeb23e2a0"
```

Comment:
You don't need to directly depend on StochasticDiffEq

Reply:
all the same, dependency on StochasticDiffEq  needed

Reply:
https://ci.appveyor.com/project/ChrisRackauckas/neuralnetdiffeq-jl/builds/26537921